ios/Extensions/DidDoc+Extensions.swift

```diff
@@ -0,0 +1,72 @@
+extension DidDoc {
+    
+    init(fromJson json: JSONDictionary) {
+        guard let did = json["did"] as? String else {
+            fatalError("Cant resolve did")
+        }
+        
+        guard let keyAgreements = json["key_agreements"] as? [String] else {
+            fatalError("Cant resolve keyAgreements")
+        }
+        
+        guard let authentications = json["authentications"] as? [String] else {
+            fatalError("Cant resolve authentications")
+        }
+        
+        
+        guard let verificationMethodsJson = json["verification_methods"] as? [JSONDictionary] else {
+            fatalError("Cant resolve verificationMethodsJson")
+        }
+        
+        guard let servicesJson = json["services"] as? [JSONDictionary] else {
+            fatalError("Cant resolve verificationMethodsJson")
```

Comment:
Error message should be `Can't resolve servicesJson`

Reply:
Also, for `DidDoc` we have less detailed errors comparing to, for example, `VerificationMethod`:
`Cant resolve verificationMethodsJson`  vs `Cant resolve verification method: id`

It's better to use more or less same error message convention where possible.